caffe2/python/onnx/backend.py

```diff
@@ -173,12 +173,29 @@ class Caffe2Backend(Backend):
         'RNN': '_create_rnn',
     }
 
+    # DummyName factory
+    @classmethod
+    def __init__(cls):
+        cls._check_dummy_name_factory()
+
+    @classmethod
+    def _check_dummy_name_factory(cls):
+        try:
+            cls._dummy_name_factory
+        except AttributeError:
+            cls._dummy_name_factory = C.DummyName()
+
+    @classmethod
+    def _dummy_name(cls):
+        dummy_name(cls._dummy_name_factory)
```

Comment:
Maybe directly use C.DummyName in backend (so no need to wrap it with dummy_name from the helper module)?

Reply:
Good idea. It does seem redundant.